reark/src/main/java/io/reark/reark/data/store/ContentProviderStore.java

```diff
@@ -60,7 +60,7 @@
     private static final String TAG = ContentProviderStore.class.getSimpleName();
 
     @NonNull
-    final protected ContentResolver contentResolver;
+    protected final ContentResolver contentResolver;
```

Comment:
This isn't actually used outside of the class in Reark. Changed it to private and added the accessor too, in case someone would need the access.